common/scripts/lint2annotations.awk

```diff
@@ -0,0 +1,12 @@
+# Converts xlint/etc format lints into GH Actions annotations
+# The original line is printed alongside the annotation command
+{
+	split($0, a, ": ")
+	split(a[1], b, ":")
+	msg = substr($0, index($0, ": ") + 2)
+	if (b[2]) {
+		line = ",line=" b[2]
+	}
+	print $0
```

Comment:
Printing error twice in logs is redundant, can we remove this?

Reply:
The current behaviour includes the file and line number in the log message, while the annotation would eat it (the file and line only being visible in the form of the inline annotation itself.

There's a couple of options
1. Remove the plain print and make the message of the annotation the same as the raw lint line (the annotation message would contain the filename [and line] even when inline)
2. Keep as is
3. Not change the annotation and remove the raw print (filename [and line] never shown anywhere)

In all 3 the inline annotations are correctly placed

Imo option 3 is fine but I essentially didn't want to "break" the current behaviour without input from reviewers. The duplicated print is easier to notice than the lack of filename[/line] in the log.

Reply:
Wow I didn't even connect that "oh nice they print the filename in green" meant that it's essentially there

Reply:
Updated and tested, the demo pr on my fork is updated too